code/game/objects/items/mail.dm

```diff
@@ -107,26 +107,46 @@
 			var/tag = uppertext(GLOB.TAGGERLOCATIONS[destination_tag.currTag])
 			to_chat(user, span_notice("*[tag]*"))
 			sort_tag = destination_tag.currTag
-			playsound(loc, 'sound/machines/twobeep_high.ogg', 100, TRUE)
+			playsound(loc, 'sound/machines/twobeep_high.ogg', vol = 100, vary = TRUE)
+
+/obj/item/mail/multitool_act(mob/living/user, obj/item/tool)
+	if(user.get_inactive_held_item() == src)
+		balloon_alert(user, "Nothing to disable!")
+		return TRUE
+	else
+		to_chat(user, span_notice("You must hold it, if you want to try disable any kind of device that might be inside!"))
+		return FALSE
+	
 
 /obj/item/mail/attack_self(mob/user)
+	if(!unwrap(user))
+		return FALSE
+	after_unwrap(user)
```

Comment:
Should we return TRUE after this? What is the return value of attack_self representing?

Reply:
yea, i guess